tests/product/test_server_install.py

```diff
@@ -227,24 +227,6 @@ def test_install(self, installer=None):
             self.assert_has_default_config(container)
             self.assert_has_default_connector(container)
 
-    def test_install_worker_is_pa_master(self):
```

Comment:
This can be removed because it's not really testing presto-admin. It's making sure that there's nothing special about the master host to server as a Presto coordinator.